nemo/core/classes/modelPT.py

```diff
@@ -447,11 +453,21 @@ def restore_from(
         """
 
         if save_restore_connector is None:
-            save_restore_connector = SaveRestoreConnector()
+            save_restore_connector = cls.get_default_save_restore_connector()
 
         if save_restore_connector.model_extracted_dir is None:
-            restore_path = os.path.abspath(os.path.expanduser(restore_path))
+            # Check if a directory was passed, and the directory contains the config file, then simply
```

Comment:
To support automatic LLM model loading without passing explicitly the `model_extracted_dir` to the SaveRestoreConnector or subclass